scripts/analysis_forecasts_init_time.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
```

Comment:
could you add a sentence or two, of what this script does?

Reply:
This script is a duplicate of the missing_dates.py script so i have removed